src/www/ui/template/browse.html.twig

```diff
@@ -54,7 +54,7 @@
               </th>
               <th><input type="checkbox" id="openCBoxFilter"/>{{ 'open'|trans }}</th>
               <th></th>
-              <th></th>
+              <th><button type="button" id="markIrrelevant" >MarkAsIrrelevant</button></th>
```

Comment:
Why not `Mark as irrelevant` with spaces?

Reply:
Because of spacing issue in action td, it increases the size of button as well as td.

Reply:
![sizeofbutton](https://user-images.githubusercontent.com/10155976/30965360-ced6fcd0-a472-11e7-8d48-ad4c6a74f3c6.PNG)